packages/fxa-auth-server/fxa-oauth-server/test/routes/token.js

```diff
@@ -230,28 +228,33 @@ describe('/token POST', function() {
     });
 
     it('allows clients that have not been disabled via config', async () => {
+      request.payload.client_id = NON_DISABLED_CLIENT_ID;
+      request.payload.grant_type = 'refresh_token';
+      request.payload.refresh_token = CODE;
       try {
         await route.handler(request);
         assert.fail('should have errored');
       } catch (err) {
-        assert.equal(err.errno, 102); // Incorrect client secret.
+        assert.equal(err.errno, 108); // Invalid token.
       }
     });
 
     it('allows code grants for clients that have been disabled via config', async () => {
       request.payload.client_id = DISABLED_CLIENT_ID;
       request.payload.grant_type = 'authorization_code';
+      request.payload.code = CODE;
       try {
         await route.handler(request);
         assert.fail('should have errored');
       } catch (err) {
-        assert.equal(err.errno, 101); // Unknown client.
+        assert.equal(err.errno, 105); // Unknown code.
```

Comment:
From the description I would have expected this test to pass. I know what you are trying to test, but the description and what's being tested don't really match.